src/external/crypto.m4

```diff
@@ -2,3 +2,25 @@ AC_DEFUN([AM_CHECK_LIBCRYPTO],
          [PKG_CHECK_MODULES([CRYPTO],[libcrypto])
           PKG_CHECK_MODULES([SSL],[libssl])
 ])
+
+AC_MSG_CHECKING([wether OpenSSL's x400Address is ASN1_STRING])
+AC_COMPILE_IFELSE(
```

Comment:
Is this reliable? It looks it depends on a compiler setting that makes it to fail on warning.

Would it make sense (if possible) to enable `AC_LANG_WERROR` explicitly for this check?

Reply:
I added `-Werror -Wall -Wextra` explicitly for this test. I think `AC_LANG_WERROR` will have an effect on all checks, so I preferred the other way.